yml/dp02_dc2.yaml

```diff
@@ -8238,3 +8238,174 @@ tables:
     datatype: long
     mysql:datatype: BIGINT
     description:
+- name: Visit
+  '@id': '#Visit'
+  description: ''
+  columns:
+  - name: visit
+    '@id': '#Visit.visit'
+    datatype: long
```

Comment:
MySQL doesn't have the numeric data type `long`. For signd 64-bit numbers you need to add the following attribute:
```
mysql:datatype: BIGINT
```
For 32-bit numbers:
```
mysql:datatype: INT
```
For unsigned types:
```
mysql:datatype: BIGINT UNSIGNED
```
or:
```
mysql:datatype: INT UNSIGNED
```